azure-pipelines/azure-templates/webhooks-template.yml

```diff
@@ -0,0 +1,23 @@
+jobs:
+  - job: DiscordMessage
+
+    steps: 
+    - task: ado-discord-webhook@1
+      inputs:
+        channelId: '856587514218545203'
+        webhookKey: 'OmVq23OAAKWh6VQGfoiFE-DRlGwtDst19OLHedR9HaQ7eONnVdlBJlc1i8OiAUpz4hqd'
+        name: 'Team III - Project III'
```

Comment:
the token is supposed to be secret and not on github, i'm pretty sure.
the other values that will certainly be different for the next iteration should not be in the templates, should be provided to the templates as parameters.

Reply:
Updated.